jvector-base/pom.xml

```diff
@@ -11,4 +11,15 @@
     </parent>
     <artifactId>jvector-base</artifactId>
```

Comment:
does this need to go in Base?

Reply:
okay, and this is a "plugin" which is distinct from a "dependency" so jvector users aren't going to pull down rat as well, is that right?

Reply:
Correct. The plugin machinery is specific to Maven workflows. It's stored in the local repository, but has nothing to do with the produced artifacts.

Reply:
Ship it!